### PR TITLE
DOCS/options: improve wording of dither-depth

### PR DESCRIPTION
**Pull request by llyyr**
and remove incorrect warning.

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -5557,13 +5557,12 @@ them.
         Automatic selection.
         On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
         On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        For ``gpu-api=vulkan``, 8 bpc is assumed for SDR content.
```
Comment (kasper93) [inline DOCS/man/options.rst:5560]: I think we should indeed simplify it. Make it more vague and say that it is best effort to provide best depth depending on vo and gpu-api. Exactly wording is up to you, but describing full logic here make it hard to parse in docs imho.

Comment (sfan5) [inline DOCS/man/options.rst:5560]: suggestion:
```
Output bit depth detection currently only works on `--gpu-api=d3d11`.
On ``--vo=gpu``: detected depth or 8 bpc otherwise
On ``--vo=gpu-next``: detected depth or 8 bpc (SDR content) or no dithering otherwise.
```

Comment (llyyr) [reply DOCS/man/options.rst]: > Output bit depth detection currently only works on `--gpu-api=d3d11`.

This is incorrect because it doesn't work for vo_gpu

Comment (sfan5) [reply DOCS/man/options.rst]: you mean because vo_gpu doesn't support d3d11? (or does it?)

Comment (llyyr) [reply DOCS/man/options.rst]: Changed

Comment (kasper93) [reply DOCS/man/options.rst]: It should work for `vo_gpu`, no?

see

https://github.com/mpv-player/mpv/blob/3e7d36c295f726e701d1ceea6af83382f7b2f59e/video/out/vo_gpu.c#L66-L66

and 
https://github.com/mpv-player/mpv/blob/3e7d36c295f726e701d1ceea6af83382f7b2f59e/video/out/gpu/video.c#L2738-L2748

Comment (llyyr) [reply DOCS/man/options.rst]: You're right, I misunderstood how it worked

File: DOCS/man/options.rst [hunk 2]
```diff
@@ -5555,15 +5555,15 @@ them.
         Disable any dithering done by mpv.
     auto
         Automatic selection.
-        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
-        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        On ``--vo=gpu``: detected depth or 8 bpc otherwise
+        On ``--vo=gpu-next``: detected depth or 8 bpc (SDR content) or no
```
Comment (kasper93) [inline DOCS/man/options.rst:5559]: Not `(SDR content)`. It is `target`. If SDR content with HDR `target-trc` would not use 8 bpc.

Comment (llyyr) [reply DOCS/man/options.rst:5559]: It should be fine no? We use the same language for --target-trc and saying SDR content should get the point across better than "for SDR target transfer"

**Comment by kasper93** [comment]
Looks ok for me now.

File: DOCS/man/options.rst [hunk 3]
```diff
@@ -5555,15 +5555,15 @@ them.
...
+        On ``--vo=gpu-next``: detected depth or 8 bpc (for SDR target) or no
+        dithering otherwise.
     8
         Dither to 8 bit output.
 
     Note that the depth of the connected video display device cannot be
-    detected. Often, LCD panels will do dithering on their own, which conflicts
-    with this option and leads to ugly output.
+    detected except when using ``gpu-api=d3d11``. Explicitly setting your
+    display's bit depth is recommended.
```
[the remark this answers is not in the document]
Comment (kasper93) [reply DOCS/man/options.rst:5566]: That's right, but applying dithering twice, is not as scary as this warning make it. It is better to dither to wire format either way. Even if there would be another dithering down the line, which we cannot really control/disable.

File: DOCS/man/options.rst [hunk 4]
```diff
@@ -5555,15 +5555,15 @@ them.
         Disable any dithering done by mpv.
     auto
         Automatic selection.
-        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
-        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        On ``--vo=gpu``: detected depth or 8 bpc otherwise
+        On ``--vo=gpu-next``: detected depth or 8 bpc (for SDR target) or no
+        dithering otherwise.
```
Comment (kasper93) [inline DOCS/man/options.rst:5560]: > or no dithering otherwise.
I would remove that. In fact gpu_next will always dither to target format, unless it is explicitly disabled with dither_depth=-1. While `detected depth` means "wire" or "format" depth.
```
    if (opts->dither_depth < 0)
        pars->params.dither_params = NULL;
```

File: DOCS/man/options.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (llyyr) [reply DOCS/man/options.rst]: > Why was the remark about LCD panel dithering deleted?

Because it was incorrect, there's no quality loss from dithering multiple times.

> It's possible for the wire format to be different from the LCD panel's capability

Changed the wording

[the remark this answers is not in the document]
Comment (kasper93) [reply DOCS/man/options.rst]: How about 
```
LCD panels may have a different actual depth compared to what they accept as input. In such situations, it's recommended to set the `dither-depth` parameter to match the panel's actual depth.
```
this value is indeed impossible to detect, even edid will not tell us true value.

[the remark this answers is not in the document]
Comment (llyyr) [reply DOCS/man/options.rst]: I can add that but I'd have thought `Explicitly setting your display's bit depth is recommended` was enough

[the remark this answers is not in the document]
Comment (kasper93) [reply DOCS/man/options.rst]: For me it is. Wait for @na-na-hi opinion.

[the remark this answers is not in the document]
Comment (llyyr) [reply DOCS/man/options.rst]: Hopefully this is good now

**Review by kasper93 — APPROVED** [review]

File: DOCS/man/options.rst (hunk 4 above)
Comment (sfan5) [reply DOCS/man/options.rst]: but what will it dither to if this is not set?
https://github.com/mpv-player/mpv/blob/374470d471a49491bceb4e1241402e3e56ccb209/video/out/vo_gpu_next.c#L850-L854

This is what made me assume that it does no dithering.

Comment (kasper93) [reply DOCS/man/options.rst]: It dithers to the target format if not disabled.
